components/Web3Btn.vue

```diff
@@ -4,10 +4,11 @@
       <v-icon>mdi-plus</v-icon>
       {{ btnAction }}
     </v-btn>
+    <h1></h1>
```

Comment:
What is this empty header about ?

Reply:
this is just me being a scrub. I'll get that removed.